tutorials/modelcard-tribuo-v4.ipynb

```diff
@@ -0,0 +1,485 @@
+    "\n",
+    "We will first re-create the anomaly detection model from the Anomaly Detection tutorial and then construct numerous `ModelCard` objects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "*\n",
+      "optimization finished, #iter = 653\n",
+      "obj = 289.5926348816893, rho = 3.144570476807895\n",
+      "nSV = 296, nBSV = 114\n",
+      "svm-anomaly-detection-model - Model(class-name=org.tribuo.common.libsvm.LibSVMModel,dataset=Dataset(class-name=org.tribuo.MutableDataset,datasource=DataSource(class-name=org.tribuo.anomaly.example.GaussianAnomalyDataSource,expectedMeans=[1.0, 2.0, 1.0, 2.0, 5.0],anomalousMeans=[-2.0, 2.0, -2.0, 2.0, -10.0],seed=1,numSamples=2000,fractionAnomalous=0.0,anomalousVariances=[1.0, 0.5, 0.25, 1.0, 0.1],expectedVariances=[1.0, 0.5, 0.25, 1.0, 0.1],host-short-name=DataSource),transformations=[],is-sequence=false,is-dense=true,num-examples=2000,num-features=5,num-outputs=2,tribuo-version=4.3.0-SNAPSHOT),trainer=Trainer(class-name=org.tribuo.anomaly.libsvm.LibSVMAnomalyTrainer,cost=1.0,coef0=0.0,seed=12345,cache_size=500.0,probability=false,nu=0.1,degree=3,eps=0.001,kernelType=RBF,p=0.1,shrinking=true,svmType=SVMType(class-name=org.tribuo.anomaly.libsvm.SVMAnomalyType,type=ONE_CLASS,host-short-name=SVMType),gamma=1.0,tribuo-version=4.3.0-SNAPSHOT,train-invocation-count=0,is-sequence=false,host-short-name=Trainer),trained-at=2022-07-22T23:22:38.566574-07:00,instance-values={},tribuo-version=4.3.0-SNAPSHOT,java-version=18.0.1.1,os-name=Mac OS X,os-arch=x86_64)\n",
+      "AnomalyEvaluation(tp=405 fp=232 tn=1363 fn=0 precision=0.635793 recall=1.000000 f1=0.777351)\n"
+     ]
+    }
+   ],
+   "source": [
+    "var trainData = new MutableDataset<>(new GaussianAnomalyDataSource(2000,0.0f, 1L));\n",
+    "var evalData = new MutableDataset<>(new GaussianAnomalyDataSource(2000,0.2f,2L));\n",
+    "var params = new SVMParameters<>(new SVMAnomalyType(SVMAnomalyType.SVMMode.ONE_CLASS), KernelType.RBF);\n",
+    "params.setGamma(1.0);\n",
+    "params.setNu(0.1);\n",
+    "\n",
+    "var trainer = new LibSVMAnomalyTrainer(params);\n",
+    "var model = trainer.train(trainData);\n",
+    "var evaluator = new AnomalyEvaluator();\n",
+    "var evaluation = evaluator.evaluate(model,evalData);\n",
+    "\n",
+    "System.out.println(model.toString());\n",
```

Comment:
Rather than print this, it might be better to print the converted provenance in json format, that way people can see how it lines up with the model card. Also no-one likes the `toString` output apart from me.

Oh, and put a couple of newlines in before printing as LibSVM insists on writing to standard out and it makes the output harder to read.